amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -441,6 +445,7 @@ type ColonyAction @model {
   type: ColonyActionType!
   blockNumber: Int!
   isMotion: Boolean
+  motionData: MotionData
```

Comment:
Why does need to be a separate type? Are we expecting it grow in field size?

Reply:
Yes it gets bigger